osu.Game.Rulesets.Tau/Skinning/Default/PlayfieldPiece.cs

```diff
@@ -55,7 +55,13 @@ public PlayfieldPiece()
         private void load(TauRulesetConfigManager config)
         {
             config?.BindWith(TauRulesetSettings.PlayfieldDim, playfieldDimLevel);
-            playfieldDimLevel.BindValueChanged(v => { background.FadeTo(v.NewValue, 100); }, true);
+            playfieldDimLevel.BindValueChanged(v => { background.FadeTo(v.NewValue, 100); });
```

Comment:
You can move this whole line to `LoadComplete`, with the immediate execution. This saves a line, and keeps things together.